facia-json/src/main/scala/com/gu/facia/client/models/Collection.scala

```diff
@@ -3,6 +3,7 @@ package com.gu.facia.client.models
 import play.api.libs.json._
 import org.joda.time.DateTime
 import play.api.libs.json.{JsValue, Json}
+import com.gu.facia.client.json.JodaFormat._
```

Comment:
Because you have imported `play-json-data`, you should be able to add 
```
import play.api.libs.json.JodaWrites._
import play.api.libs.json.JodaReads._
```
I think it provides the same as your custom one